### PR TITLE
Sapi 200 relist transactions

### PR DESCRIPTION
**Pull request by dylan-mcnamara-stubhub**
Added RelistTransaction to mirror properties in API.

### CODE REVIEW

File: src/GogoKit/Models/Response/SellerListing.cs [hunk 1]
```diff
@@ -78,6 +78,13 @@ public class SellerListing : EmbeddedSellerListing
         [Embedded("listing_notes")]
         public IList<ListingNote> ListingNotes { get; set; }
 
+        /// <summary>
+        /// The model for the transaction that this listing was created from
+        /// </summary>
+        [DataMember(Name = "relist_transaction")]
+        [Embedded("relist_transaction")]
+        public virtual RelistTransaction RelistTransaction { get; set; }
```
Comment (maxab) [inline src/GogoKit/Models/Response/SellerListing.cs:86]: I don't think we should make this virtual

File: Version.props [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>4.0.0-rc-2</VersionPrefix>
+    <VersionPrefix>4.1.0-rc-2</VersionPrefix>
```
Comment (maxab) [inline Version.props:3]: 4.0.0-rc-3

File: src/GogoKit/Models/Response/SellerListing.cs (hunk 1 above)
Comment (dylan-mcnamara-stubhub) [reply src/GogoKit/Models/Response/SellerListing.cs]: removed

File: Version.props (hunk 2 above)
Comment (dylan-mcnamara-stubhub) [reply Version.props]: updated

**Review by maxab — APPROVED** [review]

**Pull request merged**